pkg/agent/manager/sync.go

```diff
@@ -62,26 +59,37 @@ func (m *manager) synchronize(ctx context.Context) (err error) {
 			outdated++
 		default:
 			// SVID is good
-			return
+			return false
 		}
-		// we've exceeded the CSR limit, don't make any more CSRs
-		if len(csrs) < node.CSRLimit {
+
+		return true
+	})
+
+	// TODO: this values are not real, we may remove
+	if expiring > 0 {
+		telemetry_agent.AddCacheManagerExpiredSVIDsSample(m.c.Metrics, float32(expiring))
+		m.c.Log.WithField(telemetry.ExpiringSVIDs, expiring).Debug("Updating expiring SVIDs in cache")
+	}
+	if outdated > 0 {
+		telemetry_agent.AddCacheManagerOutdatedSVIDsSample(m.c.Metrics, float32(outdated))
+		m.c.Log.WithField(telemetry.OutdatedSVIDs, outdated).Debug("Updating SVIDs with outdated attributes in cache")
+	}
```

Comment:
How about we just change the language of the log line to simply declare the state instead of what we're doing with it.
```suggestion
	if expiring > 0 {
		telemetry_agent.AddCacheManagerExpiredSVIDsSample(m.c.Metrics, float32(expiring))
		m.c.Log.WithField(telemetry.ExpiringSVIDs, expiring).Debug("Cache has expiring entries")
	}
	if outdated > 0 {
		telemetry_agent.AddCacheManagerOutdatedSVIDsSample(m.c.Metrics, float32(outdated))
		m.c.Log.WithField(telemetry.OutdatedSVIDs, outdated).Debug("Cache has outdated entries")
	}
```

Reply:
done